cg/services/order_validation_service/workflows/fastq/models/sample.py

```diff
@@ -0,0 +1,14 @@
+from cg.models.orders.sample_base import PriorityEnum, SexEnum
+from cg.services.order_validation_service.constants import ElutionBuffer
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class FastqSample(Sample):
+    concentration_ng_ul: float | None = None
+    elution_buffer: ElutionBuffer
+    priority: PriorityEnum
+    quantity: int | None = None
+    require_qc_ok: bool
+    sex: SexEnum
+    source: str  # Required if no internal id
+    tumour: bool = False
```

Comment:
Not for this PR, but this reminded me that the attribute tumour seems to be incorrectly named is_tumour for the balsamic sample.

Reply:
I can change it in this PR

Reply:
I also prefer it, but I do consider renaming of variables sent to us by the FE to be out of scope and should be a separate issue/work item.

Reply:
leave tumour, fix in another PR